### ISSUE TITLE
TrenchBroomConfig static isn't ready at Startup (or PostStartup)

### ISSUE THREAD

**Issue by Manonox** [Noxmore/bevy_trenchbroom#1]
```rs
.add_plugins(TrenchBroomPlugin::new(trenchbroom_config()))
.add_systems(Startup, spawn_test_map)
```

# Expected
Should spawn a map, duh

# Result
```
thread 'IO Task Pool (1)' panicked at C:\Users\Manonox\.cargo\registry\src\index.crates.io-6f17d22bba15001f\bevy_trenchbroom-0.2.0\src\util.rs:17:31:
No TrenchBroomConfig mirrored, please add a TrenchBroomPlugin to your app.
```

# Description
https://github.com/Noxmore/bevy_trenchbroom/blob/fbd08e3b44c34679259e1faa168542afe47216eb/src/lib.rs#L39
This is only added to PreUpdate, which probably results in the inability to spawn maps at Startup/PostStartup

**Comment by Manonox** [comment]
A possible workaround:
```rs
.add_systems(PreStartup, mirror_trenchbroom_config)
```

**Comment by Noxmore** [comment]
Whoops! Should be fixed in 0.2.1.
